.github/workflows/main.yml

```diff
@@ -348,14 +348,14 @@ jobs:
         if: ${{ success() && (github.event_name != 'pull_request' || env.SEEKING_TESTERS == '1') }}
         run: |
           CI/windows/03_package_obs.ps1 -BuildArch ${{ matrix.arch }} -Package
-          $ArtifactName = Get-ChildItem -filter "OBS-Studio-*-Win-${{ matrix.arch }}.zip" -File
+          $ArtifactName = Get-ChildItem -filter "obs-studio-windows-*-${{ matrix.arch }}.zip" -File
```

Comment:
This line needs to be changed to:
```
$ArtifactName = Get-ChildItem -filter "obs-studio-*-windows-${{ matrix.arch }}.zip" -File
```

The pattern here comes from `DestinationPath` in `CI/windows/03_package_obs.ps1`:
```
    $FileName = "obs-studio-${VersionString}-windows"
```
```
            DestinationPath = "${FileName}-x64.zip"
```
So `DestinationPath` is: `obs-studio-${VersionString}-windows-${{ matrix.arch }}.zip`.

Reply:
Fixed